### DISCUSSION TITLE
Unable to run Fennec scheme in simulator on M2 laptop

### DISCUSSION THREAD

**Discussion by mleers** [mozilla-mobile/firefox-ios#13225] [Q&A] [answered]
Hi,

I am new to the community and hoping to contribute.  I am currently running into issues running Fennec on the simulator after successfully building in Xcode (14.1) on a 2022 M2 MacBook Air.

After building, the simulator pops up and tries to launch Fennec but fails with the following error:

```
Details

Application launch for 'org.mozilla.ios.Fennec' did not return a valid pid nor a launch error.
Domain: NSPOSIXErrorDomain
Code: 3
Failure Reason: No such process
```

After following the Readme (including building user scripts, resetting and resolving packages, again on a fresh download) I have tried the following in some combination:
 - Replaced Team under Signing & Capabilities with my own personal team for all Targets
 - Replaced DEVELOPMENT_TEAM in Common.xcconfig with my own team id
 - Cleaning project
 - Restarting simulator and making sure simulator isn't running before Xcode
 - ctrl-f the mozilla team code (43AQ936H96) in another code editor and replace with my own elsewhere in the project like [this](https://github.com/mozilla-mobile/firefox-ios/discussions/8788) thread suggests
 - Excluded architecture for Fennec set to arm64 (default setting)

I have a hunch that it is related to architecture - has anyone that uses Apple silicon run into this before?

Thanks

**Comment by lmarceau** [comment]
Hi @mleers, we have some devs over here running on M1 and recently have a dev that started building with an M2 with success. @nbhasin2 did you ran into any of the issues listed up here? In case you can add onto the thread to debug this. Thanks!

**Comment by nbhasin2** [comment]
There should be no issues on M1 or M2 mac with XCode 14.1 as I run on a M2 mac
The XCode 14.2 has the black screen issue for WKWebview but since you are on 14.1 that isn't a problem.

**Comment by mleers** [comment, marked as answer]
@lmarceau @nbhasin2 I found the fix [here](https://stackoverflow.com/a/65744082).  Essentially, Xcode needs to be launched using Rosetta (and installed first if needed like in my case).
